<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeToLabelClass</class>
 <widget class="QMainWindow" name="VolumeToLabelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VolumeToLabel</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiel">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="action_vifo"/>
     <addaction name="action_json"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="action_net"/>
     <addaction name="action_node"/>
     <addaction name="action_csv"/>
    </widget>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
   </widget>
   <addaction name="menuFiel"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_vifo">
   <property name="text">
    <string>*.vifo</string>
   </property>
  </action>
  <action name="action_net">
   <property name="text">
    <string>*.net</string>
   </property>
  </action>
  <action name="action_node">
   <property name="text">
    <string>*.node</string>
   </property>
  </action>
  <action name="action_json">
   <property name="text">
    <string>*.json</string>
   </property>
  </action>
  <action name="action_csv">
   <property name="text">
    <string>*.csv</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VolumeToLabel.qrc"/>
 </resources>
 <connections/>
</ui>
